deploy/ansible/inventory/group_vars/iris_servers.yml

```
@@ -0,0 +1,61 @@
+---
+# Variables specific to IRIS servers
+
+# IRIS Application Configuration
+iris_server_name: "dfir-iris.local"
+iris_app_version: "v2.4.20"
+iris_https_port: 443
+
+# Database Configuration
+postgres_user: "postgres"
+postgres_admin_user: "iris_admin" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)